ee/src/main/java/org/jboss/as/ee/concurrent/ManagedScheduledExecutorServiceImpl.java

```diff
@@ -38,7 +38,7 @@
 import static org.jboss.as.ee.concurrent.ControlPointUtils.doWrap;
 
 /**
- * WildFly's extension of Java EE 7 RI {@link org.glassfish.enterprise.concurrent.ManagedScheduledExecutorServiceImpl}.
+ * WildFly's extension of Jakarta EE 7 RI {@link org.glassfish.enterprise.concurrent.ManagedScheduledExecutorServiceImpl}.
```

Comment:
The version here should likely be removed.

Reply:
I think that we can also remove Jakarta, will change to:
>  WildFly's extension of {@link org.glassfish.enterprise.concurrent.ManagedScheduledExecutorServiceImpl}.